src/server.js

```diff
@@ -18,6 +18,7 @@ db.once('open', () => console.log(`Connected to database`));
 
 // middleware
 app.use(express.json());
+app.use(morgan('dev'));
```

Comment:
This is causing an error since the morgan package isn't a dependency in `package.json`.
Either add as a dependency and import in `server.js`, or delete this line. We can add morgan later if need be.

Reply:
Oh and one last thing, merge the `develop` branch into this branch.

Reply:
Hello Shane,  

Thank you for review.
Now I have removed morgan middleware from server.js file and added event router to the routes array.